mtgjson5/resources/booster_box_size_overrides.json

```diff
@@ -34,7 +34,8 @@
         "CLB": 18
     },
     "collector": {
-        "2X2": 6
+        "2X2": 6,
+        "MAT": 12
```

Comment:
@kodabb - i closed this as I'm obviously lacking context about how specific things are used. However I would suspect this item is still needed as it seems similar to the 2X2 situation.

Reply:
unless I'm mistaken 12 is the default value for collector boosters, but the "default" one does need to be set to 24, will you send a pr for it?

Reply:
Oh yeah that's correct - the number of packs by default for collector boxes is 12.

However the number of cards per pack is what's odd about MAT, I must have been thinking that.
Normal collector boosters have 15 cards, MAT only have 6 cards per pack.